### PR TITLE
feat(spans): Add metric strings for mobile

### PR DESCRIPTION
**Pull request by jjbayer**
As a follow-up to https://github.com/getsentry/relay/pull/2379 and in preparation for future PRs around mobile span metrics, add corresponding strings to the static indexer lookup.

### CODE REVIEW

File: src/sentry/sentry_metrics/indexer/strings.py [hunk 1]
```diff
@@ -150,6 +150,9 @@
     "span.group": PREFIX + 252,
     "transaction.method": PREFIX + 253,
     "span.category": PREFIX + 254,
+    "span.contribution": PREFIX + 255,
+    "span.main_thread": PREFIX + 256,
+    "device.class": PREFIX + 257,
```
Comment (jjbayer) [inline src/sentry/sentry_metrics/indexer/strings.py:155]: I was not able to check these strings against `sentry_stringindexer` and `sentry_perfstringindexer` yet. Blocking merge until we are sure that these strings are not already in use.

Comment (jjbayer) [reply src/sentry/sentry_metrics/indexer/strings.py:155]: Now verified.

**Review by shruthilayaj — APPROVED** [review]
🚢

File: src/sentry/sentry_metrics/indexer/strings.py [hunk 2]
```diff
@@ -150,6 +150,9 @@
     "span.group": PREFIX + 252,
     "transaction.method": PREFIX + 253,
     "span.category": PREFIX + 254,
+    "span.contribution": PREFIX + 255,
```
Comment (shruthilayaj) [inline src/sentry/sentry_metrics/indexer/strings.py:153]: hmm, actually @nikhars left this comment on my data requests [doc](https://www.notion.so/sentry/span-contribution-9b5a0979a81645789db6877593729cf4?pvs=4)
> Tag values are always stored as strings in the metrics platform. Also, we should avoid having multiple values in a single tag key. So I would second Abhi’s suggestion to have different tag key like is_ttid and is_ttfd

So should we not add this for now?

Comment (jjbayer) [inline src/sentry/sentry_metrics/indexer/strings.py:153]:
```suggestion
```

Comment (jjbayer) [reply src/sentry/sentry_metrics/indexer/strings.py:153]: OK, removed for now.

**Pull request merged**